src/ts/.eslintrc

```diff
@@ -0,0 +1,32 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true,
+        "commonjs": true
+    },
+    "parser": "@typescript-eslint/parser",
+    "parserOptions":{
+        "sourceType": "module",
+        "project": "./tsconfig.json"
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/recommended"
+    ],
```

Comment:
List of ESLint Rules:
[eslint:recommend](https://github.com/eslint/eslint/blob/master/conf/eslint-recommended.js)
[typescript-eslint/recommend](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/src/configs/recommended.json)